.github/fabricbot.json

```diff
@@ -1,3600 +0,0 @@
-        "label": "auto merge :zap:",
-        "silentMode": false,
-        "minMinutesOpen": "480",
-        "mergeType": "merge",
-        "allowAutoMergeInstructionsWithoutLabel": false,
-        "deleteBranches": true,
-        "enforceDMPAsStatus": true,
-        "requireAllStatuses": false,
-        "requireAllStatuses_exemptList": [],
-        "removeLabelOnPush": true,
-        "conditionalMergeTypes": [],
-        "requireSpecificCheckRuns": false,
-        "usePrDescriptionAsCommitMessage": true,
-        "requireSpecificCheckRunsList": [
-          "Toolkit-CI",
-          "WIP",
-          "license/cla"
-        ]
-      },
-      "id": "NWRMh_TRrV",
-      "disabled": false
-    },
-    {
-      "taskType": "trigger",
-      "capabilityId": "IssueResponder",
-      "subCapability": "IssueCommentResponder",
-      "version": "1.0",
-      "config": {
-        "conditions": {
-          "operator": "and",
-          "operands": [
-            {
-              "operator": "not",
-              "operands": [
-                {
-                  "name": "isOpen",
-                  "parameters": {}
-                }
-              ]
-            },
-            {
-              "name": "isAction",
-              "parameters": {
-                "action": "created"
-              }
-            },
-            {
-              "name": "hasLabel",
-              "parameters": {
-                "label": "no-recent-activity :chart_with_downwards_trend:"
-              }
-            },
-            {
-              "name": "hasLabel",
-              "parameters": {
-                "label": "needs author feedback :memo:"
-              }
-            },
-            {
-              "operator": "not",
-              "operands": [
-                {
-                  "name": "noActivitySince",
-                  "parameters": {
-                    "days": 7
-                  }
-                }
-              ]
-            },
-            {
-              "operator": "not",
-              "operands": [
-                {
-                  "name": "isCloseAndComment",
-                  "parameters": {}
-                }
-              ]
-            },
-            {
-              "name": "isActivitySender",
-              "parameters": {
-                "user": {
-                  "type": "author"
-                }
-              }
-            },
-            {
-              "operator": "not",
-              "operands": [
-                {
-                  "name": "activitySenderHasPermissions",
-                  "parameters": {
-                    "permissions": "none"
-                  }
-                }
-              ]
-            }
-          ]
-        },
-        "eventType": "issue",
-        "eventNames": [
-          "issue_comment"
-        ],
-        "taskName": "For issues closed due to inactivity, re-open an issue if issue author posts a reply within 7 days.",
```

Comment:
This task didn't get ported over because event triggers don't support `noActivitySince` in the new service. /cc @mkArtakMSFT